client/src/charts/wrapped_nivo/wrapped_nivo_bar.js

```diff
@@ -42,6 +42,7 @@ const bar_table = (
     <SmartDisplayTable
       data={_.map(data, (row) => _.pick(row, [indexBy, ...keys]))}
       column_configs={column_configs}
+      unsorted_initial={true}
```

Comment:
Should `unsorted_initial={true}` just be the default?

Reply:
At this point, it may as well